packages/safe-core-sdk/src/utils/transactions/gas.ts

```diff
@@ -15,24 +15,24 @@ function estimateDataGasCosts(data: any): number {
   return data.match(/.{2}/g).reduce(reducer, 0)
 }
 
-export async function estimateTxGas(
-  contract: GnosisSafe,
+export const estimateTxGas = async (
```

Comment:
this was changed to be an arrow function but the function `estimateGasForTransactionExecution` below and `estimateDataGasCosts` above is defined with `function` keyword, I think it's better to stay consistent